services/main-frontend/src/components/page-specific/manage/courses/id/pages/NewCourseModuleForm.tsx

```diff
@@ -35,7 +42,17 @@ const NewCourseModuleForm: React.FC<Props> = ({ chapters, onSubmitForm }) => {
     },
   })
 
-  const onSubmitFormWrapper = (fields: Fields) => {
+  const onSubmitFormWrapper = async (fields: Fields) => {
+    if (fields.ects_credits) {
```

Comment:
move this to the useToastMutation in the parent component

Reply:
Maybe could we merge this update to the endpoint that the useToastMutation in the parent component already uses?